package.json

```diff
@@ -46,8 +46,9 @@
     ]
   },
   "scripts": {
-    "start": "cross-env NODE_ENV=development ./node_modules/.bin/webpack --config webpack.config.js --watch",
-    "build": "webpack --config webpack.config.js",
+    "start": "cross-env NODE_ENV=development ./node_modules/.bin/webpack --config webpack.config.js",
```

Comment:
Could we make simple - `start` and `build` command?
start is prepare env and tools for developing (watching, server).
build is make build version of sdk.